azure-login/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "$AZURE_SERVICE_APP_ID" ]] && [[ -n "$AZURE_SERVICE_PASSWORD" ]] && [[ -n "$AZURE_SERVICE_TENANT" ]]
+then
+  az login --service-principal --username "$AZURE_SERVICE_APP_ID" --password "$AZURE_SERVICE_PASSWORD" --tenant "$AZURE_SERVICE_TENANT"
+else
+  echo "One of the required parameters for Azure Login is not set: AZURE_SERVICE_APP_ID, AZURE_SERVICE_PASSWORD, AZURE_SERVICE_TENANT."
+  exit 1
+fi
+
+SUBSCRIPTIONS=$(az account list)

```

Comment:
Should you get  the list ony if $Azure_subscription is not provided

Reply:
got it.